### PR TITLE
Fixed bugs on Wiki and resolved #667

### LINKED ISSUE #667: Load wiki page failed
- Gitea version (or commit ref): 1.0.0+116-g69b86378
- Git version: 
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

https://try.gitea.io/lunny/vscode/wiki/Code-Editor-Roadmap
The wiki page cannot be loaded but https://github.com/Microsoft/vscode/wiki/Code-Editor-Roadmap is OK.
...

### PR DESCRIPTION
**Pull request by lunny**
The previous version didn't diff `wiki title` and `wiki page path`. This PR will diff them clearly in variable names.

### CODE REVIEW

File: models/wiki.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func discardLocalWikiChanges(localPath string) error {
 }
 
 // updateWikiPage adds new page to repository wiki.
-func (repo *Repository) updateWikiPage(doer *User, oldTitle, title, content, message string, isNew bool) (err error) {
+func (repo *Repository) updateWikiPage(doer *User, oldWikiPath, wikiPath, content, message string, isNew bool) (err error) {
```
Comment (Bwko) [inline models/wiki.go:92]: Why did you change this?

Comment (lunny) [reply models/wiki.go:92]: Delete wiki by path not the title. we have to give the real meaning of the variables.

Comment (Bwko) [reply models/wiki.go:92]: But now `wikiPath` contains the title of the page and not the `/path/to/wiki/page`

Comment (lunny) [reply models/wiki.go:92]: No. `wikiPath` will not contain title. See `AddWikiPage` and `EditWikiPage` which will call the function.

Comment (Bwko) [reply models/wiki.go:92]: Now I see, I created single word wiki pages. When you do that it looks like title == wikiPath

**Comment by bkcsoft** [comment]
I've looked through it, didn't see any errors 🎉 
If you've tested this you have my LGTM 😄

**Comment by Bwko** [comment]
LGTM

**Pull request merged**